Apps/WebClient/src/NewClientApp/src/components/public/terms-of-service/TermsOfServiceView.vue

```diff
@@ -0,0 +1,80 @@
+<script setup lang="ts">
+import { onMounted, ref } from "vue";
+
+import HtmlTextAreaComponent from "@/components/common/HtmlTextAreaComponent.vue";
+import LoadingComponent from "@/components/common/LoadingComponent.vue";
+import PageTitleComponent from "@/components/common/PageTitleComponent.vue";
+import BreadcrumbComponent from "@/components/site/BreadcrumbComponent.vue";
+import { container } from "@/ioc/container";
+import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
+import BreadcrumbItem from "@/models/breadcrumbItem";
+import { ResultError } from "@/models/errors";
+import { ILogger, IUserProfileService } from "@/services/interfaces";
+import { useErrorStore } from "@/stores/error";
+
+const breadcrumbItems: BreadcrumbItem[] = [
+    {
+        text: "Terms of Service",
+        to: "/termsOfService",
+        active: true,
+        dataTestId: "breadcrumb-terms-of-service",
+    },
+];
+
+const logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
+const userProfileService = container.get<IUserProfileService>(
+    SERVICE_IDENTIFIER.UserProfileService
+);
+
+const errorStore = useErrorStore();
+
+const isLoading = ref(true);
+const hasErrors = ref(false);
+const errorMessage = ref("");
+const termsOfService = ref("");
+
+function loadTermsOfService(): void {
+    isLoading.value = true;
```

Comment:
This function is only called once, and `isLoading` is already initialized to true above.  You could potentially get rid of the function (and `isLoading.value = true`) and call `userProfileService.getTermsOfService()` directly at the end of the setup.